opentelemetry/proto/logs/v1/logs.proto

```diff
@@ -165,13 +165,27 @@ message LogRecord {
 
   // A unique identifier for a trace. All logs from the same trace share
   // the same `trace_id`. The ID is a 16-byte array. An ID with all zeroes
-  // is considered invalid. Can be set for logs that are part of request processing
-  // and have an assigned trace id. [Optional].
+  // is considered invalid.
+  //
+  // This field is optional.
+  //
+  // The receivers SHOULD assume that the log record is not associated with a
+  // trace if any of the following is true:
+  //   - the field is not present,
+  //   - the field is equal to zero-length bytes sequence (or to empty string in OTLP/JSON),
+  //   - the field contains an invalid value.
```

Comment:
L175 seems to be subsumed by L176, since zero-length is also invalid. 

Also, rather than explaining invalid here I would pull it into L168: "An ID with all zeroes OR of length other than 16 bytes is considered invalid."

Reply:
Reworded, PTAL.